sympy/matrices/solvers.py

```diff
@@ -699,6 +708,67 @@ def _pinv_solve(M, B, arbitrary_matrix=None):
             A_pinv.multiply(A)).multiply(arbitrary_matrix)
 
 
+def _cramer_solve(M, rhs, det_method="bird"):
+    """Solves system of linear equations using Cramer's rule.
+
+    This method is relatively inefficient compared to other methods.
+    However it only uses a single division, assuming a division free
+    determinant method is provided.
```

Comment:
```suggestion
    determinant method is provided. This is helpful to minimize the chance of divide-by-zero cases in symbolic solutions to linear systems.
```

or something like that.

Reply:
Addressed in a777540c78ce8da5ca9e68d4182f68b6d299a231